packages/actor-query-operation-orderby-sparqlee/lib/ActorQueryOperationOrderBySparqlee.ts

```diff
@@ -65,12 +66,7 @@ export class ActorQueryOperationOrderBySparqlee extends ActorQueryOperationTyped
 
       // Sort the annoted stream
       const sortedStream = new SortIterator(transformedStream, (a, b) => {
-        const orderA = termToString(a.result);
-        const orderB = termToString(b.result);
-        if (!orderA || !orderB || orderA === orderB) {
-          return 0;
-        }
-        return orderA > orderB === isAscending ? 1 : -1;
+        return orderTypes(a.result, b.result, isAscending);
```

Comment:
Great!
Can even be simplified more by inlining: (`new SortIterator(transformedStream, (a, b) => orderTypes(a.result, b.result, isAscending))`)